examples/http-client-oauth2-jwt-bearer-grant-type/http_client_oauth2_jwt_bearer_grant_type.md

```diff
@@ -1,6 +1,6 @@
 # HTTP client - OAuth2 JWT bearer grant type
 
-A client, which is secured with an OAuth2 JWT bearer grant type can be used to connect to a secured service. The client is enriched with the `Authorization: Bearer <token>` header by passing the `http:OAuth2JwtBearerGrantConfig` to the `auth` configuration of the client.
+The `http:Client` can be secured with OAuth2 JWT bearer grant type authorization by enriching each request from the client with the `Authorization: Bearer <token>` header. The required configurations for this grant type can be specified in the `auth` field of the client configuration. Use this grant type when the client wants to receive access tokens without transmitting sensitive information such as the client secret.
```

Comment:
Lets remove this part `Use this grant type when the client wants to receive access tokens without transmitting sensitive information such as the client secret`

Reply:
Fixed via https://github.com/ballerina-platform/ballerina-distribution/pull/3793/commits/373fd382554d45179c4ec44e8ea2280b4b8709b2